### PR TITLE
Remove invisible unicode characters

### PR DESCRIPTION
**Pull request by nehalrp**

### CODE REVIEW

**Comment by nehalrp** [comment]
Working on other potential edge cases...

File: spec/functions/files/test_printable [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Test Credentials
+TEST_USER=test+user@nextdoor.com
+TEST_PASSWORD=somebadpassword
+TEST_API_KEY=asdvadjv7dsa7v56sdv7sdv7s7dvds6v90asc0
+
+# Test Key
+TEST_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----
```
Comment (stlava) [inline spec/functions/files/test_printable]: Where's this from?

Comment (nehalrp) [inline spec/functions/files/test_printable]: From googling `sample rsa private key`

File: spec/functions/files/test_input_invisible [hunk 2]
```diff
@@ -0,0 +1,5 @@
+TEST_1='A᠎B'
```
Comment (stlava) [inline spec/functions/files/test_input_invisible]: Can you add a comment above each test string with the special char that's being tested. Otherwise looks good!

```
# Test 1 : U+180E
TEST_1='A᠎B'
...
```
